### PR TITLE
Expansion panel for mobile breakpoint ready

### PR DESCRIPTION
**Pull request by pcabreram1234**

### CODE REVIEW

**Review by itsalb3rt — CHANGES_REQUESTED** [review]
Great job, but please check your changes, the pull request has 10 modified files 👀

**Comment by itsalb3rt** [comment]
This PR has an issue ? Is related with the #18 ?

**Comment by pcabreram1234** [comment]
Yes, it is to resolve the isse described previously.

**Comment by pcabreram1234** [comment]
Great thanks for the recomendation sensei

File: src/views/products/add.vue [hunk 1]
```diff
@@ -149,6 +148,9 @@ export default {
 						this.$store.commit('products/ADD', response.data.data);
 						this.name = '';
 						this.price = 1;
+						this.brand="";
```
Comment (itsalb3rt) [inline src/views/products/add.vue:151]: Don't make a `brand`, `measurementUnit` and `category` blank after save an product, this dispatch the form validation 
image

File: src/components/shop/ShopResumeMobile.vue [hunk 2]
```diff
@@ -0,0 +1,91 @@
+<template>
```
Comment (itsalb3rt) [inline src/components/shop/ShopResumeMobile.vue:1]: The idea is so great but the new component is out of the view port 
image

You can make this float and make a sure you test this in multiple screen height

File: src/views/shop/registrationShopping.vue [hunk 3]
```diff
@@ -168,6 +169,9 @@ export default {
 				item => item.id_product === idProduct
 			);
 			this.$store.commit('shoppingCar/REMOVE', indexInStore);
+		},
```
Comment (itsalb3rt) [inline src/views/shop/registrationShopping.vue:172]: This method is never declared but is invoked

File: src/views/shop/shoppingCar.vue [hunk 4]
```diff
@@ -235,4 +235,11 @@ export default {
 	margin-bottom: 150px;
 	padding-bottom: 20px;
 }
+
```
Comment (itsalb3rt) [inline src/views/shop/shoppingCar.vue:238]: Why this modification ? This is not a part of the main issue

**Review by itsalb3rt — CHANGES_REQUESTED** [review]

File: src/views/products/add.vue (hunk 1 above)
Comment (pcabreram1234) [reply src/views/products/add.vue:151]: This is done

File: src/components/shop/ShopResumeMobile.vue (hunk 2 above)
Comment (pcabreram1234) [reply src/components/shop/ShopResumeMobile.vue:1]: I just resolved it

File: src/views/shop/registrationShopping.vue (hunk 3 above)
Comment (pcabreram1234) [reply src/views/shop/registrationShopping.vue:172]: I do not know, i didn't change the original code of this lines

File: src/views/shop/shoppingCar.vue (hunk 4 above)
Comment (pcabreram1234) [reply src/views/shop/shoppingCar.vue:238]: Thanks for your advice, I just remove this lines, it was affecting the padding and margin of the products container in the shopping car

Comment (itsalb3rt) [reply src/views/shop/shoppingCar.vue:238]: Maybe this can be a new issue 😃

File: src/views/shop/registrationShopping.vue (hunk 3 above)
Comment (pcabreram1234) [reply src/views/shop/registrationShopping.vue:172]: It is possible

File: src/views/shop/ShopResumeMobile.vue [hunk 5]
```diff
@@ -0,0 +1,91 @@
+<template>
```
Comment (itsalb3rt) [inline src/views/shop/ShopResumeMobile.vue:1]: This file is duplicate with the `src/components/shop/ShopResumeMobile.vue` ?

**Review by itsalb3rt — APPROVED** [review]
Great work Phillip! Thanks for the collaboration 🙌

**Pull request merged**